lib/elements/steel_reinforcement/base.rb

```diff
@@ -0,0 +1,14 @@
+module Elements
+  class Base
```

Comment:
I think you are missing here a `module SteelReinforcement` between `Elements` and `Base`

Reply:
Oh right!